README.md

```diff
@@ -15,7 +15,8 @@
 ## Compatibility
 
 | PHPStan version | Yii2 extension version |
-| --------------- | ---------------------- |
+|-----------------|------------------------|
+| 1.0             | 1.x                    |
```

Comment:
I would rather not bump this extension version to 1.x 🤔

Reply:
Done